src/vtbackend/Terminal.h

```diff
@@ -631,7 +631,7 @@ class Terminal
 #if defined(__APPLE__) || defined(_MSC_VER)
         reply(fmt::vformat(message, fmt::make_format_args(args...)));
 #else
-        reply(fmt::vformat(message, fmt::make_format_args(std::forward<Ts>(args)...)));
+        reply(fmt::vformat(message, fmt::make_format_args(args...)));
```

Comment:
that is interesting. I thought it's usually a good idea to do perfect forwarding. but here it maybe not the case. hmmm. ok

Reply:
So the idea is that we want to prevent rvalue reference there, so that is why we can not forward things there anymore, few issues from fmt repo: https://github.com/fmtlib/fmt/issues/3443 , https://github.com/fmtlib/fmt/issues/3589

Reply:
ok thx